FEXCore/Source/Interface/Core/CPUBackend.h

```diff
@@ -115,7 +115,7 @@ namespace CPU {
       uint16_t HostPCOffset;
 
       // How much to offset the RIP from the previous entry.
-      uint16_t GuestRIPOffset;
+      int16_t GuestRIPOffset;
```

Comment:
Should we add an assert in JIT.cpp to ensure this never overflows? Seems a bit sketchy to have this 16-bit limitation with seemingly no safeguards in place to ensure we consistently adhere to it.